docs/ibc/overview.md

```diff
@@ -6,170 +6,205 @@ parent:
+implement callbacks that are called during the channel handshake. These callbacks can do custom
+channel initialization logic. If any callback returns an error, the channel handshake fails. Thus, by
+returning errors on callbacks, modules can programmatically reject and accept channels.
 
-The channel handshake is a 4 step handshake. Briefly, if a given chain A wants to open a channel with
+The channel handshake is a 4-step handshake. Briefly, if a given chain A wants to open a channel with
 chain B using an already established connection:
 
 1. chain A sends a `ChanOpenInit` message to signal a channel initialization attempt with chain B.
 2. chain B sends a `ChanOpenTry` message to try opening the channel on chain A.
 3. chain A sends a `ChanOpenAck` message to mark its channel end status as open.
 4. chain B sends a `ChanOpenConfirm` message to mark its channel end status as open.
 
-If all this happens successfully, the channel will be open on both sides. At each step in the handshake, the module
-associated with the `ChannelEnd` will have it's callback executed for that step of the handshake. So
-on `ChanOpenInit`, the module on chain A will have its callback `OnChanOpenInit` executed.
+If all handshake steps are successful, the channel is opened on both sides. At each step in the handshake, the module
+associated with the `ChannelEnd` executes its callback. So
+on `ChanOpenInit`, the module on chain A executes its callback `OnChanOpenInit`.
 
-Just as ports came with dynamic capabilites, channel initialization will return a dynamic capability
+Just as ports came with dynamic capabilities, channel initialization returns a dynamic capability
 that the module **must** claim so that they can pass in a capability to authenticate channel actions
 like sending packets. The channel capability is passed into the callback on the first parts of the
 handshake; either `OnChanOpenInit` on the initializing chain or `OnChanOpenTry` on the other chain.
 
 ### [Packets](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel)
 
-Modules communicate with each other by sending packets over IBC channels. As mentioned above, all
+Modules communicate with each other by sending packets over IBC channels. All
 IBC packets contain the destination `portID` and `channelID` along with the source `portID` and
-`channelID`, this allows modules to know the sender module of a given packet. IBC packets also
-contain a sequence to optionally enforce ordering. IBC packets also contain a `TimeoutTimestamp` and
-`TimeoutHeight`, which when non-zero, will determine the deadline before which the receiving module
-must process a packet. If the timeout passes without the packet being successfully received, the
-sending module can timeout the packet and take appropriate actions.
+`channelID`. This packet structure allows modules to know the sender module of a given packet. IBC packets 
+contain a sequence to optionally enforce ordering. 
```

Comment:
so the packet sequence that defines ordering is 
  `sequence: uint64` ?
  seems like we would want to define what is ORDERED and what is UNORDERED

Reply:
Yes if a channel is ORDERED, then the packets must be received in the order of the `sequence` that is inside the packet.